java/tools/pom.xml

```diff
@@ -57,13 +57,13 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <version>${hadoop.version}</version>
+      <version>${tools.hadoop.version}</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-hdfs</artifactId>
-      <version>${hadoop.version}</version>
+      <artifactId>hadoop-hdfs-client</artifactId>
```

Comment:
For Apache Spark 2.10, we don't need to change this dependency.
For Apache Spark 3.2.2, we need this change.

Reply:
Tests seem to pass for 3.2.2 -- I am not sure we should have an explicit hadoop version for tool though.
Any reason we are not bumping hadoop.version project wide? We could still keep the min supported version to 2.2 right?

```
    <min.hadoop.version>2.2.0</min.hadoop.version>
    <hadoop.version>2.7.3</hadoop.version>
    <tools.hadoop.version>3.2.2</tools.hadoop.version>
```

Reply:
Do you think it's safe? Of course, we should bump `hadoop.version` later, but I don't think that is trivial like this, @pgaref .

Hadoop 3 has many breaking changes like https://github.com/apache/orc/pull/608 is also a behavior change from Hadoop layer. This PR is designed by @omalley 's decision. He explicitly mentioned `tools` jar's Hadoop version, not the all Hadoop version.
> We should upgrade the version of Hadoop used for the tools jar to the higher 2.7.7 from 2.7.3.

I believe we can upgrade `tools` first and do the main later with more testing.

Reply:
Thanks for clarifying Dongjoon! 
Since in this ticket we are targeting the specific issue mentioned by Owen I would vote for bumping to 2.10 which is more clean. In a follow-up we target Hadoop 3 project-wise. What do you think?

Reply:
No problem. Thank YOU, @pgaref . What I wanted was your real feedback here. :) I'll move back to `2.10.x`.